behaviours/meme_rater.py

```diff
@@ -276,9 +277,22 @@ async def delete_meme(event: hikari.GuildReactionAddEvent) -> None:
     if not shit_reaction:
         return
 
-    count = shit_reaction.count
+    # Find the "🔟" reaction; if not found then return 0.
+    ten_reaction = next(
+        (reaction for reaction in message.reactions if reaction.emoji == "🔟"), 0
+    )
+
+    # Convert to int var
+    ten_reaction_count = (
+        ten_reaction.count if isinstance(ten_reaction, hikari.messages.Reaction) else 0
```

Comment:
```suggestion
        ten_reaction.count if ten_reaction else 0
```

Reply:
And then you can inline it into the subtraction below

Reply:
I'm assuming you want `else 0` changed back to `else None` then

Reply:
Your change alone won't work; prefer to leave as is

Reply:
The lion doesn't concern himself with else Nones